packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -392,15 +399,18 @@ export interface IGarbageCollectorConfigs {
 	readonly sessionExpiryTimeoutMs: number | undefined;
 	/** The time after which an unreferenced node is ready to be swept. */
 	readonly sweepTimeoutMs: number | undefined;
+	/** The delay between tombstone and sweep. Not persisted. */
```

Comment:
Since this mentions "Not persisted", it should always say what the implications are, i.e., it can change across sessions.